### DISCUSSION TITLE
Is there any way to have a different dynamic route before [locale]?

### DISCUSSION THREAD

**Discussion by gajda-w** [amannn/next-intl#880] [Q&A] [answered]
I'm building an application where the requirement is for the link to look like the following: domain.com/[region]/[language]. However, when I add another dynamic route above the [locale] route in my current implementation, the language is taken from the highest dynamic route, which is [region]. This is quite a non-standard configuration; I know that standardly it might be something like en-US, but is there any option to achieve it the way I described?

Can I somehow instruct next-intl to use the values in the [locale] route rather than the one above it?

**Comment by amannn** [comment, marked as answer]
You're probably looking for this: https://github.com/amannn/next-intl/issues/653

**Reply by gajda-w** [reply]
Looks great, thank you! 🙏
